packages/exchange/src/exchange/providers/anthropic.py

```diff
@@ -1,4 +1,5 @@
 import os
+from typing import Tuple
```

Comment:
these are [deprecated](https://docs.python.org/3/library/typing.html#deprecated-aliases), we should use `tuple` directly